<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor de Correo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Welcome">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>¡Bienvenido a tu bandeja de entrada!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="New">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Correo.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Read">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Leer Correo.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Modify">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar Correo.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar Correo.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Security">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Copias de Seguridad.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Propietario">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>450</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Formato Propietario.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FastSearch">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Búsqueda Rápida.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dispersion">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>590</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Dispersión y Serialización.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>660</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir del Programa.</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_write">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_read">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_modify">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_delete">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_security">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_propietario">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>460</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_fast">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_dispersion">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>590</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_exit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>670</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
